src/components/Dropdown/Dropdown.tsx

```diff
@@ -35,11 +35,14 @@ interface Props {
 const Dropdown = (props: Props) => {
   const { text, size, id, items, direction, variant, style, alignRight } = props
 
+  const noop = () => {}
+
   const getDropdownItem = (item: Item, i: number) => (
     <DropdownRB.Item
       style={item.style}
       key={item.key || `dropdown-item-${i.toString()}`}
       eventKey={item.eventKey || `dropdown-event-${i.toString()}`}
+      onClick={item.onClick || noop}
```

Comment:
I'm not sure I understand the need for a `noop` function. Could you explain?

Reply:
I think this might be due to lack of understanding of how javascript works. I put the noop function in the case that item.onClick is null (just passes an empty function otherwise).

Reply:
Removed noop. It's not needed here.